sections/adguard-specific/ads.txt

```diff
@@ -12,10 +12,9 @@ myonlineradio.hu#$#._bannerTop1 { background-color: transparent !important; }
 
 myonlineradio.hu#$#.topHorizontalBanner { position: absolute !important; left: -3000px !important; }
 
-! https://github.com/hufilter/hufilter-dev/issues/440
+! https://github.com/hufilter/hufilter-dev/pulls/452
 !#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
-mobilarena.hu###page-forum #center > div:has(h4 + [onclick*="https://mobilarena.hu/muvelet/hozzaferes/kapcsolodas.php"])
-prohardver.hu###page-forum #center > div:has(h4 + [onclick*="https://prohardver.hu/muvelet/hozzaferes/kapcsolodas.php"])
+prohardver.hu,mobilarena.hu,itcafe.hu,gamepod.hu,logout.hu,hardverapro.hu##div:has(>h4 + div[style*="calc"])
```

Comment:
please add this rule to the `sections/ads.txt` too, but with `-abp-has`,
also, please put a space after the combinator (`>`)

Reply:
It's already in ads.txt

https://github.com/hufilter/hufilter-dev/blob/4d7354ef955611282b916b779b7cdcd29f15c549/sections/ads.txt#L1197